.github/actions/rust-test/action.yaml

```diff
@@ -64,9 +60,6 @@ runs:
         # This is required to run some JNI related tests on the Rust side
         JAVA_LD_LIBRARY_PATH=$JAVA_HOME/lib/server:$JAVA_HOME/lib:$JAVA_HOME/lib/jli
         # special handing for java 1.8 for both linux and mac distributions
-        if [ $JAVA_VERSION == "8" ]; then
```

Comment:
I wonder why this is no longer required

Reply:
With Java11 the tests run without any LD_LIBRARY_PATH setting since the upgrade to jni-rs 0.21

Reply:
Cool, good to know!